bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/io/ChannelPoolTest.java

```diff
@@ -31,35 +33,62 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
-import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
 
+import com.google.cloud.bigtable.grpc.io.ChannelPool.WrappedChannel;
+
 
 @RunWith(JUnit4.class)
 @SuppressWarnings({"rawtypes", "unchecked"})
 public class ChannelPoolTest {
 
-  @Mock
-  private Channel channel;
-  @Mock
-  private MethodDescriptor descriptor;
-  @Mock
-  private ClientCall callStub;
-  @Mock
-  private ClientCall.Listener responseListenerStub;
-  @Mock
-  private HeaderInterceptor interceptor;
-
   @Test
   public void testInterceptorIsCalled() throws Exception {
-    MockitoAnnotations.initMocks(this);
-    when(channel.newCall(any(MethodDescriptor.class), any(CallOptions.class))).thenReturn(
+    Channel channel = mock(Channel.class);
```

Comment:
There are different requirements for mocking objects in different test cases.  We can either mock all combinations for every test, or specify the objects we want to mock for a specific test case.  I opted for the latter.